typings/tests.ts

```diff
@@ -1,4 +1,4 @@
-import { APIInteractionGuildMember } from 'discord-api-types';
+import { APIGuildMember, APIInteractionGuildMember, APIMessage } from 'discord-api-types';
```

Comment:
Why is this not versioned 😢 
I know it falls outside the scope of this PR but please version this